sdk/src/main/java/velocitas/Utils.kt

```diff
@@ -0,0 +1,71 @@
+package velocitas
```

Comment:
package name to be discussed. 

I could not find anything comparably in the cpp one, only the namespace "velocitas": https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/sdk/include/sdk/grpc/AsyncGrpcFacade.h#L24

Reply:
decision: org.eclipse.velocitas.sdk